backend/Social/SocialAPI.py

```diff
@@ -491,8 +547,8 @@ def getOfficialPosts():
             item['name'] = userIDtoName(item.get('userID'))
             ccaID = int(item.get('ccaID'))
             profile = db.Profiles.find_one({'userID': item.get('userID')})
-            item['profilePictureURI'] = profile.get(
-                'profilePictureUrl') if profile != None else None
+            item['imageKey'] = profile.get(
```

Comment:
Don't understand what is going on here. Don't you need to use S3.read() to retrieve the signedurl and add that to the response?